packages/utils/ts-utils/src/loadTSConfig.js

```diff
@@ -21,7 +21,7 @@ export async function loadTSConfig(config: Config, options: PluginOptions) {
   let parsedConfig = ts.parseJsonConfigFileContent(
     configResult,
     host,
-    path.dirname(nullthrows(config.resolvedPath)),
+    nullthrows(config.rootDir),
```

Comment:
Not to self: this is failing tests as this should be the tsconfig file path.

Should `config.getConfig` return something like `{content: any, filePath: string}` or something?